pkg/session_recording/recordings.jsx

```diff
@@ -249,7 +254,7 @@
                     <div className="row">
                         <div className="col-md-12">
                             <ol className="breadcrumb">
-                                <li><a>Session Recording</a></li>
+                                <li><a onClick={this.goBackToList}>Session Recording</a></li>
```

Comment:
Ha-ha, nice. Thanks, it would be embarrassing if the resize got stuck on the demo at FrOSCon.